perfkitbenchmarker/pkb.py

```diff
@@ -257,6 +260,9 @@ def RunBenchmark(benchmark, collector, sequence_number, total_benchmarks,
     benchmark_config: The config to run the benchmark with.
     benchmark_uid: An identifier unique to this run of the benchmark even
       if the same benchmark is run multiple times with different configs.
+
+  Returns:
+    True.
```

Comment:
In order to support a request in #573, I added an option to suppress exceptions to RunParallelThreads, in which case the returned list, which would otherwise be a list of return values from the called functions, is filled in with None for functions that failed. There are other ways around it (e.g. defining a new sentinel value to put in the list or changing the shape of the returned object if suppress_exceptions is set). At the moment, this True is here because it's not None.